sshnp_docker_demo/db-template.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#
+# DO NOT RUN THIS SCRIPT. THIS IS MEANT TO BE RUN BY ANOTHER SCRIPT IN ONE OF THE THREE DIRECTORIES
+#
+
+if [ -z "$1" ]
+then
+    echo "Usage: ./db.sh <sshnp/sshnpd/sshrvd>"
+    exit 1
+fi
+
```

Comment:
Test for presence of .atKeys, otherwise stuff is going to fail later.

Reply:
Done in https://github.com/atsign-foundation/at_demos/pull/97/commits/d516cd0281c05c34ec133573f91aee656f611ec7